images/spire/config/main.tf

```diff
@@ -46,6 +50,15 @@ output "config" {
       command = var.entrypoint-cmd
     }
     cmd = var.cmd
+    paths = [
+      {
+        path      = "/certs"
+        type      = "directory"
+        uid       = local.spire-user
```

Comment:
You should be able to use `module.accts.block.run-as` here rather than defining a local

[Example](https://github.com/chainguard-images/images/blob/main/images/kube-logging-operator-fluentd/config/main.tf#L53)